### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#502] [Esami & Esoneri M-Z]
### Traccia

<img width="945" alt="image" src="https://github.com/user-attachments/assets/0cdd3a91-00f1-4bd4-9e29-d891357385c9">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/2a652842-b7bd-40f3-bfa7-f2f37efcaa9c)
